<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdaptiveFilterForm</class>
 <widget class="QWidget" name="AdaptiveFilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="moveRightButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>781</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adaptive Filter Form</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="moveLeftButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="longLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Long avg: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>% Diff: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="shortLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Short avg: 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shortMask">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="difference">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Min Diff: 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="differenceMask">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="longMask">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="percentMask">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>moveRightButton</zorder>
  <zorder>moveLeftButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>longLabel</zorder>
  <zorder>percent</zorder>
  <zorder>shortLabel</zorder>
  <zorder>shortMask</zorder>
  <zorder>difference</zorder>
  <zorder>differenceMask</zorder>
  <zorder>longMask</zorder>
  <zorder>percentMask</zorder>
  <zorder>saveButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
